src/DataTable.php

```diff
@@ -567,7 +576,13 @@ public function itemToArray($item): array
         $itemArray = $item->toArray();
         $preserved = [];
         foreach ($returnKeys as $key) {
-            $value = $itemArray[$key] ?? false;
+            $value = data_get($itemArray, $key, '__no_data__');
```

Comment:
I'm not sure if I'm a fan of hardcoded string as default. Would prefer a unique safe string aka Str::uuid()->toString()